code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -376,6 +376,19 @@
 	M.radiation += 4
 	..()
 
+/datum/reagent/toxin/magillitis
```

Comment:
It doesn't really matter much. But being under the toxin path just makes it count for liver shenanigans and trekchem anti-toxin. Which seems pointless when it just makes monkeys into gorillas.